pcsx2/PAD/Host/KeyStatus.cpp

```diff
@@ -104,9 +115,9 @@ void KeyStatus::Set(u32 pad, u32 index, float value)
 			m_analog[pad].ry = m_analog[pad].invert_ry ? MERGE(pad, PAD_R_UP, PAD_R_DOWN) : MERGE(pad, PAD_R_DOWN, PAD_R_UP);
 		}
 #undef MERGE
-
+#if 0
```

Comment:
Was this intended to get left in?

Reply:
Yeah, I thought it was a good reference to do circle things later on.